getting-started/clis/calicoctl/install.md

```diff
@@ -305,6 +305,8 @@ docker pull {{page.registry}}{{page.imageNames["calicoctl"]}}:{{site.data.versio
 
 
 Use the YAML that matches your datastore type to deploy the `calicoctl` container to your nodes.
+Determine datastore type using `kubectl exec -ti -n kube-system calicoctl -- calicoctl version` 
```

Comment:
I think we have an ordering problem here! The command here won't work until you run one of the commands below :)

Reply:
You could do this, though:

```
kubectl get configmap -n kube-system calico-config -o yaml | grep etcd
```

If that returns "etcd", you are using etcd mode. Otherwise, it's not.

Reply:
It might be better to just do this:

```
kubectl get clusterinformation -o yaml | grep clusterType
```

Reply:
Thanks for pointing out this problem! The instructions mandate the use of YAML that matches the datastore type. It seems like there should be a definitive, concise way to determine this and include in the instructions. 
I am a beginner just trying to improve the documentation based upon the challenges and questions I faced.

Neither the configmap or the clusterinformation seem to work for my environment:

`kubectl get pod --namespace=kube-system`
 `kubectl describe pod etcd-k8y --namespace=kube-system | grep Image:`
  Image:         k8s.gcr.io/etcd:3.4.13-0

Any suggestions appreciated.

Reply:
> Neither the configmap or the clusterinformation seem to work for my environment:

What do they return? I'm surprised they don't work.